test/foo/lib/config/test_configloader.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from foo.lib.config import ConfigLoader, ConfigLoaderException
+
+
+def test_load(monkeypatch):
```

Comment:
Also, it's not a problem yet but if the ConfigLoader was created before in the same context but for example with additional defaults this test will fail.